### PR TITLE
Bump cairo lib to v0.16.0

### PR DESCRIPTION
**Pull request by andrew-fleming**

### CODE REVIEW

File: packages/core-cairo/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - **Breaking changes**:
   - Use OpenZeppelin Contracts for Cairo v0.15.0. ([#378](https://github.com/OpenZeppelin/contracts-wizard/pull/378))
+  - Use OpenZeppelin Contracts for Cairo v0.16.0. ([#384](https://github.com/OpenZeppelin/contracts-wizard/pull/384))
```
Comment (immrsd) [inline packages/core-cairo/CHANGELOG.md:7]: Should we keep the previous change mentioning `0.15.0`?

File: packages/core-cairo/test_project/Scarb.toml [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
-cairo-version = "2.7.0"
-scarb-version = "2.7.0"
+cairo-version = "2.8.0"
+scarb-version = "2.8.0"
 
 [dependencies]
-starknet = "2.7.0"
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.15.0" }
+starknet = "2.8.0"
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.16.0" }
```
Comment (immrsd) [inline packages/core-cairo/test_project/Scarb.toml:10]: Can we specify it as `openzeppelin = "0.16.0`?

File: packages/core-cairo/CHANGELOG.md (hunk 1 above)
Comment (andrew-fleming) [reply packages/core-cairo/CHANGELOG.md:7]: I don't think we should overwrite changes in the changelog. I'd keep it in

File: packages/core-cairo/test_project/Scarb.toml (hunk 2 above)
Comment (ericglau) [reply packages/core-cairo/test_project/Scarb.toml]: Should the readme steps on https://github.com/OpenZeppelin/cairo-contracts also use that format?

**Review by ericglau — APPROVED** [review]

**Pull request merged**